lib/inputstreamhelper.py

```diff
@@ -462,21 +497,26 @@ def _latest_widevine_version(self, eula=False):
             versions = self._http_get().decode()
             return versions.split()[-1]
 
-        return [x for x in self._chromeos_config() if config.CHROMEOS_ARM_HWID in x['hwidmatch']][0]['version']
+        devices = self._chromeos_config()
+        arm_device = self._select_best_chromeos_image(devices)
+        return arm_device['version']
```

Comment:
If there was no  Chrome OS image found, we get `NoneType' object has no attribute '__getitem__`

Reply:
Correct, at the moment there is no support for the case where there is no match. And we probably should, however. Before this change we only selected on a single HWID, without error-handling. And now we have the possibility of 20 supported HWIDs.

So I think this PR improves it to the point that not finding a matching HWID is improbable. But I agree we should take into account other causes (like the file being corrupt, empty, etc.)